static-sample/js/create-kink-load.js

```diff
@@ -0,0 +1,42 @@
+$(document).ready(function() {
+				row = $("<tr></tr>");
```

Comment:
oh crap, I see what you mean now. I forgot to `git rm -r` that stuff during the rebase. Forget this PR then. I'll rebase, squash the commits, and make it all a little cleaner.